Slim/Utils/Errno.pm

```diff
@@ -39,11 +39,19 @@ our @EXPORT = qw(EWOULDBLOCK EINPROGRESS EINTR ECHILD EBADF);
 
 BEGIN {
         if (main::ISWINDOWS) {
-                *EINTR       = sub () { 10004 };
-                *EBADF       = sub () { 10009 };
-                *ECHILD      = sub () { 10010 };
-                *EWOULDBLOCK = sub () { 10035 };
-                *EINPROGRESS = sub () { 10036 };
+		if (main::ISACTIVEPERL) {
+			*EINTR       = sub () { 10004 };
+			*EBADF       = sub () { 10009 };
+			*ECHILD      = sub () { 10010 };
+			*EWOULDBLOCK = sub () { 10035 };
+			*EINPROGRESS = sub () { 10036 };
+		} else {
+			*EINTR       = sub () { 4 };
+			*EBADF       = sub () { 9 };
+			*ECHILD      = sub () { 10 };
+			*EWOULDBLOCK = sub () { 140 };
+			*EINPROGRESS = sub () { 112 };
```

Comment:
Are these really different from the POSIX values? Or is this a mistake?

According to https://perldoc.perl.org/Errno.txt:

```
EWOULDBLOCK => 11,
EINPROGRESS => 115,
```

Reply:
Ok, that's probably a difficult topic. Found eg. https://www.nntp.perl.org/group/perl.perl5.porters/2015/03/msg226651.html

Never mind.

Reply:
If those defines were wrong for strawberry perl, then LMS wouldn't work as both are referenced in the code base.  I spent a lot of time researching this when I first tried to get LMS working with SBP.   I had to add a block of code to display the values associated with those defines when I was troubleshooting why LMS locked up when started with SBP.  So at least for SBP v5.32 they are correct.